RELEASE.md

```
@@ -0,0 +1,71 @@
+# How to release a new version of Holidays
+
+-   Finalize the current development version
+    -   switch to `dev` branch and pull the most recent changes from
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)